.github/policies/msgraph-sdk-php-branch-protection.yml

```diff
@@ -33,6 +33,9 @@ configuration:
     requiresConversationResolution: true
     # Are merge commits prohibited from being pushed to this branch. boolean
     requiresLinearHistory: false
+    # Required status checks to pass before merging. Values can be any string, but if the value does not correspond to any existing status check, the status check will be stuck on pending for status since nothing exists to push an actual status
+    requiredStatusChecks:
+    - validate-pull-request
```

Comment:
Current status checks fail due to missing custom Model types. Priority was to fix this in the Kiota version.
Is it possible for this change be shelved until the Kiota version is GA?